src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -78,7 +78,7 @@ var ContainerSettingsComponent = React.createClass({
         <div className="col-sm-3">
           <FormGroupComponent
             attribute={`container.docker.portMappings[${i}].containerPort`}
-            label="Container Port"
+            label="Container port"
```

Comment:
I know this is not correct, but in the sake of layout we are capitalizing our words.
```Port``` should be written with a capital letter.
I think ssk2 ment this:
```
Container path
```
becomes
```
Container Path
```